### PR TITLE
Profiles: route qr code to profile sheet

### PR DESCRIPTION
**Pull request by benisgold**
Fixes TEAM2-409
Figma link (if any):

## What changed (plus any additional context for devs)
- route qr code to profile sheet

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Comment by benisgold** [comment]
pending TF build for testing https://app.bitrise.io/build/387e9545-032e-48e2-afad-a24539df3add

File: src/hooks/useScanner.ts [hunk 1]
```diff
@@ -57,14 +59,18 @@ export default function useScanner(enabled: any, onSuccess: any) {
 
       // And then navigate to Showcase sheet
       InteractionManager.runAfterInteractions(() => {
-        Navigation.handleAction(Routes.SHOWCASE_SHEET, {
-          address,
-        });
+        Navigation.handleAction(
+          profilesEnabled ? Routes.PROFILE_SHEET : Routes.SHOWCASE_SHEET,
```
Comment (estebanmino) [inline src/hooks/useScanner.ts:63]: this is ok but we also want it on line 76 `handleScanRainbowProfile`

**Comment by benisgold** [comment]
https://app.bitrise.io/build/387e9545-032e-48e2-afad-a24539df3add

**Review by jxom — APPROVED** [review]
lgtm

**Review by estebanmino — CHANGES_REQUESTED** [review]
i think we need to resolve the name here if is scanning the address https://www.loom.com/share/3478a882e5c5423f90e1771612a29142

**Review by estebanmino — APPROVED** [review]
https://www.loom.com/share/ae25d82ceb044767bfa03ed7ba1c8ad2 resolving names before navigating

**Review by jxom — APPROVED** [review]
lgtm!

**Pull request merged**